docs(k8s): update quickstart (#1752)
scaleway/docs-content: containers/kubernetes/quickstart.mdx

### How to add external nodes to a multi-cloud pool

1. Click **Kubernetes** in the **Containers** section of the side menu. The Kubernetes dashboard displays.
2. Click <Icon name="more" /> next to the cluster you want to edit. Click **More info** in the menu. The cluster dashboard displays. Click the **Pools** tab. 
3. Click <Icon name="more" /> next to the pool you want to edit. Click **More info** in the menu. Click **Add node** in the menu. The node information page displays. 
4. Connect to your node using SSH and run the commands displayed on your screen after you have added the node.
   ```sh 
    wget https://scwcontainermulticloud.s3.fr-par.scw.cloud/multicloud-init.sh && chmod +x multicloud-init.sh
    ./multicloud-init.sh -p c58bcee3-52db-4501-a96e-41dddbb6e5f4 -r fr-par -t SCW_SECRET_KEY
    ```
  <Message type="important">
    Make sure you have: 
    * An `arm64` or `amd64` server running with Ubuntu or Debian, having a public IP assigned to it
    * A Scaleway [secret key](/console/my-project/how-to/generate-api-key)
  </Message>

>   </Message type="tip">

Reviewer: @bene2k1 le slash est en trop ça fait une erreur

### How to add external nodes to a multi-cloud pool

1. Click **Kubernetes** in the **Containers** section of the side menu. The Kubernetes dashboard displays.

> 

Author: ```suggestion
```

1. Click **Kubernetes** in the **Containers** section of the side menu. The Kubernetes dashboard displays.
2. Click <Icon name="more" /> next to the cluster you want to edit. Click **More info** in the menu. The cluster dashboard displays. Click the **Pools** tab. 
3. Click <Icon name="more" /> next to the pool you want to edit. Click **More info** in the menu. Click **Add node** in the menu. The node information page displays. 
4. Connect to your node using SSH and run the commands displayed on your screen after you have added the node.

  <Message type="important">
    Make sure you have: 

>     * A server running with Ubuntu 20.04, having a public IP assigned to it

Reviewer 2: ```suggestion
    * A arm64 or amd64 server running with Ubuntu or Debian, having a public IP assigned to it
```

>     * An arm64 or amd64 server running with Ubuntu or Debian, having a public IP assigned to it

Author: ```suggestion
    * An `arm64` or `amd64` server running with Ubuntu or Debian, having a public IP assigned to it
```
